bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```diff
@@ -565,6 +565,11 @@ data:
         admin_network_policy_enabled_flag="--enable-admin-network-policy"
       fi
 
+      dns_name_resolver_enabled_flag=
```

Comment:
`start-ovnkube-node` is only used by ovnkube-node, this means that the `ovnkube-cluster-manager` won't rollout if `DNS_NAME_RESOLVER_ENABLE` changes.

Reply:
I think it will, as it uses `004-config.yaml` file for features setup

Reply:
Lest consider a scenario in which the featuregate is enabled during cluster runtime(e.g cluster is moved to techpreview):
1. FeatureGate gets enabled
2. CNO restarts
3. CNO renders the new 008-script-lib.yaml and 004-config.yaml confimaps
4. ovnkube-node daemonset is rolled out because it is annotated with the has of the script: https://github.com/openshift/cluster-network-operator/blob/6aa2f0fbf2b06ce52db1a2a039bbdbcaf588c409/pkg/network/ovn_kubernetes.go#L356-L378
5. ovnkube-controller is not rolled out and it doesn't reload the configmap dynamically on its own. 

does this make sense?

Reply:
we agreed it is a bigger problem and created a card for it https://issues.redhat.com/browse/SDN-4832